configure.ac

```diff
@@ -682,12 +682,12 @@ fi
 dnl ----- Mac OSX specific -----
 if test x"$this_os" = "xmacosx"; then 
 	AC_MSG_RESULT([ * Mac OSX specific configuration])
-	AC_DEFINE(BSD4_4, 1, [BSD compatiblity macro])
+	dnl AC_DEFINE(BSD4_4, 1, [BSD compatiblity macro])
 	AC_DEFINE(HAVE_2ARG_DBTOB, 1, [Define if dbtob takes two arguments])
 	dnl AC_DEFINE(NO_DLFCN_H)
-	AC_DEFINE(NO_DDP, 1, [Define if DDP should be disabled])
+	dnl AC_DEFINE(NO_DDP, 1, [Define if DDP should be disabled])
 	AC_DEFINE(NO_QUOTA_SUPPORT, 1, [Define if Quota support should be disabled])
-	AC_DEFINE(MACOSX_SERVER, 1, [Define if compiling for MacOS X Server])
+	dnl AC_DEFINE(MACOSX_SERVER, 1, [Define if compiling for MacOS X Server])
```

Comment:
Nitpick: What's your reason for commenting out these lines over removing them altogether? It might be worth leaving a comment why they're commented out at least. :)

Reply:
Shall I remove all MACOSX_SERVER conditionals from the codebase, it makes more sense if we're removing OS X Server host support?

Reply:
I thought it might be worth leaving them in in case they are required by a future userland AppleTalk network stack??